db/schema.go

```diff
@@ -82,7 +82,6 @@ func Prepare(conn *pgx.Conn, schemas []string, includeTables, excludeTables []st
 	}
 
 	for _, table := range registerTables {
-		log.Infof("registering trigger for table %s", table)
```

Comment:
Can we keep this logging statement? It's useful to know what's happening/ where we are in the setup? Maybe the message can be "Verifying and registering trigger" ?

Reply:
The output is misleading. It can be changed, but it's the only log line in the whole process.

Reply:
Plus its using the standard log package, not logrus. Logging needs to be fixed up across the whole project, but not something I want to start tackling before database migrations (if it can be avoided).